tests/bsim/bluetooth/ll/bis/src/main.c

```diff
@@ -147,6 +147,40 @@ bool ll_data_path_sink_create(uint16_t handle, struct ll_iso_datapath *datapath,
 }
 #endif /* CONFIG_BT_CTLR_ISO_VENDOR_DATA_PATH */
 
+#define BUF_ALLOC_TIMEOUT   (30) /* milliseconds */
+NET_BUF_POOL_FIXED_DEFINE(tx_pool, CONFIG_BT_ISO_TX_BUF_COUNT,
+			  BT_ISO_SDU_BUF_SIZE(CONFIG_BT_ISO_TX_MTU), 8, NULL);
+
+static struct k_work_delayable iso_send_work;
+
+static void iso_send(struct k_work *work)
+{
+	uint8_t iso_data[CONFIG_BT_ISO_TX_MTU] = { 0, };
+	struct net_buf *buf;
+	int ret;
+
+	buf = net_buf_alloc(&tx_pool,
+			    K_MSEC(BUF_ALLOC_TIMEOUT));
+	if (!buf) {
+		FAIL("Data buffer allocate timeout on channel\n");
+		return;
+	}
+	net_buf_reserve(buf, BT_ISO_CHAN_SEND_RESERVE);
+	sys_put_le32(seq_num, iso_data);
```

Comment:
I guess this assumes that `CONFIG_BT_ISO_TX_MTU >= 4` - Should we add a BUILD_ASSERT or something for that?